Gemfile.jruby-2.5.lock.release

```diff
@@ -545,7 +549,7 @@ GEM
       sequel
       tzinfo
       tzinfo-data
-    logstash-integration-kafka (10.8.2-java)
+    logstash-integration-kafka (10.9.0-java)
```

Comment:
some updates (e.g. the Kafka one) are not related to log4j, but I was planning to update plugins as they contain useful ECS related updates, thus :+1: